src/MatBlazor.Demo/Pages/Index.razor

```diff
@@ -134,7 +134,7 @@
 
     public void Dispose()
     {
-        Console.WriteLine("My Dispose 2");
+        //Console.WriteLine("My Dispose 2");
```

Comment:
Can we just remove the code rather than comment it out? I'm not certain the comment provides much of anything moving forward.

Reply:
done